src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -228,28 +205,36 @@ const TravelAidSection = ({
               {isCancelled && <CancelledDepartureMessage />}
 
               {situationsForFocused.map((situation) => (
-                <SituationMessageBox key={situation.id} situation={situation} />
+                <SituationMessageBox
+                  key={situation.id}
+                  situation={situation}
+                  a11yAnnounce={true}
+                />
               ))}
 
               {noticesForFocused.map(
                 (notice) =>
                   notice.text && (
-                    <MessageInfoBox type="info" message={notice.text} />
+                    <MessageInfoBox
+                      type="info"
+                      message={notice.text}
+                      a11yAnnounce={true}
+                    />
                   ),
               )}
             </View>
           )}
-          <View
-            accessible
-            accessibilityLabel={focussedStateA11yLabel}
-            style={styles.stopHeaderContainer}
-          >
-            <View>
+          <View accessible style={styles.stopHeaderContainer}>
+            <LiveRegionWrapper
+              accessibilityLabel={
+                getStopHeader(status, t) + screenReaderPause + quayName
+              }
+            >
```

Comment:
Should there be two layers of `accessible` here?

Reply:
The topmost `accessible` here is to better match the sketches, where the time and stop place is one focus area. It is, however, handled differently on iOS and Android. Talkback makes the two children focusable, while VoiceOver focuses the parent. Both platforms have the same announcements. I've clarified with @hannahATB that it's ok to have it this way.